### PR TITLE
Fix 0002-add-vsc8531-ethernet.patch

### PR DESCRIPTION
**Pull request by matinlotfali**
I changed the patch so it would only affect one file and one devicetree node only.
Worked fine on V2L repo

### CODE REVIEW

**Review by mistyron — COMMENTED** [review]
Did you get a chance to load this image and test out the ethernet interfaces?

**Comment by matinlotfali** [comment]
@mistyron  I confirm that the latest changes work correctly on a G2L.
It compiles successfully for a V2L but a working V2L is not tested yet.

**Comment by matinlotfali** [comment]
Tested and it also works on V2L

**Review by mistyron — APPROVED** [review]
been tested on and appears to work, LGTM!

File: recipes-kernel/linux/dts/add-vsc8531-ethernet.dts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#include <dt-bindings/net/mscc-phy-vsc8531.h>
+
+&eth0 {
+        phy0: ethernet-phy@7 {
```
Comment (mistyron) [inline recipes-kernel/linux/dts/add-vsc8531-ethernet.dts:4]: while I approve this patch to get it into our device tree & working, we really should change this label to ` ethernet-phy@0` but when I tried it previously, I got some conflicts, there might need to be another change too. I've not looked into it yet.

Comment (mistyron) [reply recipes-kernel/linux/dts/add-vsc8531-ethernet.dts:4]: The number behind the `@` is the MDIO address

**Pull request merged**